content/scala-cli.md

```diff
@@ -627,46 +627,23 @@ Using directives can be place on only top of the file (above imports, package de
 Comments containing directives needs to start by `>`, for example:
 
 ```
-//> using Scala 3
-/*> using 
-  Scala 3
-  option "-Xfatal-warning"
-*/
+//> using scala 3
+//> using platform scala-js
+//> using options -Xasync
 ```
 
 We propose following sytax for Using Directives (within special comments described above):
 
 ```
-UsingDirective ::= "using" (Setting | Settings)
-Settings ::= "{" Setting { ";" Setting } [";"] "}"
-Setting ::= Ident [Values | Settings]
-Ident ::= ScalaIdent { “.” ScalaIdent }
-Values ::= Value { "," Value} [","]
-Value ::= stringLiteral | ["-"] numericLiteral | true | false
+UsingDirective ::= "using" Setting
+Setting ::= Ident ( Value | Values )
+Ident ::= ScalaIdent { "." ScalaIdent }
+Values ::= Value { ", " Values }
```

Comment:
The coma is optional now, so we should probably put it into [], the space is not however.